pypeit/core/wavecal/templates.py

```diff
@@ -767,23 +767,72 @@ def main(flg):
 
     # P200 DBSP r
     if flg & (2 ** 30):
-        # HeNeAr
+        # HeNeAr 316/7500 D55
```

Comment:
we've stopped using this approach to running the code in this
module.  If you have the stomach for it, please follow one of the
examples in the `spectrographs` folder under `wavecal`

Reply:
I've created a `templ_p200_dbsp.py` file and put the code to create DBSP wavelength templates in there. Should I remove the old code in `templates.py` for creating the DBSP template spectra?

Reply:
yes, please